### PR TITLE
serverless: Support TiFlash system tables

### PR DESCRIPTION
**Pull request by breezewish**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)  -- TiDB Cloud Doc is referencing this version
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/limited-sql-features.md [hunk 1]
```diff
@@ -97,8 +97,6 @@ TiDB Cloud works with almost all workloads that TiDB supports, but there are som
 | `information_schema` | `TIDB_HOT_REGIONS` | Not supported [^4] | Not supported [^4] |
 | `information_schema` | `TIDB_HOT_REGIONS_HISTORY` | Supported | Not supported [^1] |
 | `information_schema` | `TIDB_SERVERS_INFO` | Supported | Not supported [^1] |
-| `information_schema` | `TIFLASH_SEGMENTS` | Supported | Not supported [^1] |
-| `information_schema` | `TIFLASH_TABLES` | Supported | Not supported [^1] |
```
Comment (Oreoxmt) [inline tidb-cloud/limited-sql-features.md:101]: @breezewish Please invite a tech reviewer and remove the following notes:

https://github.com/pingcap/docs/blob/c4cb48307cc7a9b7580aa01445a4d848a0c0ac81/information-schema/information-schema-tiflash-tables.md?plain=1#L11-L14

https://github.com/pingcap/docs/blob/c4cb48307cc7a9b7580aa01445a4d848a0c0ac81/information-schema/information-schema-tiflash-segments.md?plain=1#L11-L14

Comment (breezewish) [reply tidb-cloud/limited-sql-features.md:101]: Thank you! Nice catch.

**Review by Oreoxmt — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Review by baiyuqing — APPROVED** [review]

**Comment by breezewish** [comment]
@Oreoxmt This PR could be merged now, thanks!

**Pull request merged**